torch/onnx/_internal/fx/exporter.py

```diff
@@ -35,7 +35,14 @@ def _export(
     decomposed_module = passes.Decompose(
         module, export_options.decomposition_table, export_options.enable_dynamic_axes
     ).run(*args)
-
+    # Run FakeTensorProp to get fixed shape of nodes for op_level_debug purposes.
+    # NOTE: torch.fx.Transformer makes a copy of the graph only but shares weights
+    # with the original module.
+    static_reference_graph_module = torch.fx.Transformer(decomposed_module).transform()
+    static_reference_graph_module = passes.ShapeInferenceWithFakeTensor(
+        static_reference_graph_module
+    ).run(*args)
+    export_options.update(static_reference_graph=static_reference_graph_module.graph)
```

Comment:
Should not stash arbitrary things under `export_options`. With new exporter api this should presumably live under `DynamoExporter`.

Reply:
Sure, I can change this to a passing argument

Reply:
Done